pkg/xbvr/task_content.go

```diff
@@ -122,51 +181,46 @@ func ScrapeJAVR(queryString string) {
 		}
 
 	}
-	RemoveLock("scrape")
+	models.RemoveLock("scrape")
 }
 
 func ExportBundle() {
-	if !CheckLock("scrape") {
-		CreateLock("scrape")
+	if !models.CheckLock("scrape") {
+		models.CreateLock("scrape")
+		t0 := time.Now()
 
 		tlog := log.WithField("task", "scrape")
 		tlog.Info("Exporting content bundle...")
 
 		var knownScenes []string
-		var collectedScenes []scrape.ScrapedScene
+		collectedScenes := make(chan models.ScrapedScene, 100)
 
-		scrapers := scrape.GetScrapers()
+		var scrapedScenes []models.ScrapedScene
+		go sceneSliceAppender(&scrapedScenes, collectedScenes)
 
-		if len(scrapers) > 0 {
-			for _, scraper := range scrapers {
-				tlog.Infof("Scraping %s", scraper.Name)
-				scraper.Scrape(knownScenes, &collectedScenes)
-			}
-		} else {
-			tlog.Info("No scrapers registered!")
-		}
+		runScrapers(knownScenes, false, false, collectedScenes)
 
 		out := ContentBundle{
 			Timestamp:     time.Now().UTC(),
 			BundleVersion: "1",
-			Scenes:        collectedScenes,
+			Scenes:        scrapedScenes,
 		}
 
 		content, err := json.MarshalIndent(out, "", " ")
 		if err == nil {
-			fName := filepath.Join(appDir, fmt.Sprintf("content-bundle-%v.json", time.Now().Unix()))
+			fName := filepath.Join(common.AppDir, fmt.Sprintf("content-bundle-%v.json", time.Now().Unix()))
 			err = ioutil.WriteFile(fName, content, 0644)
 			if err == nil {
-				tlog.Infof("Export complete, file saved to %v", fName)
+				tlog.Infof("Export completed in %v, file saved to %v", time.Now().Sub(t0), fName)
```

Comment:
Can we ISO-format the date here?

Reply:
It's not a date.. it's a duration i.e. 3m30s.